src/styles/app.scss

```diff
@@ -1 +1,7 @@
 // Plik styli aplikacji
+@import "variables";
+
+#geo-app{
+  min-height: 100vh;
+  background-image: $gradient_background;
```

Comment:
Ja bym dorzuciła jeszcze background-repeat: no-repeat oraz cover, żeby dobrze się zawsze wszystko wyświetlało.

Reply:
Ale to jest gradient i będzie dobrze się wyświetlał, nie wiem czy to coś zmieni

Reply:
skupiłam się na tym że to bck-image. Jak to jest gradient to faktycznie nic nie da :)